controllers/alerts/alerts.go

```diff
@@ -27,6 +27,7 @@ const (
 	outOfBandUpdateAlert          = "KubevirtHyperconvergedClusterOperatorCRModification"
 	unsafeModificationAlert       = "KubevirtHyperconvergedClusterOperatorUSModification"
 	installationNotCompletedAlert = "KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert"
+	singleStackIPv6Alert          = "KubevirtHCOSingleStackIPv6"
```

Comment:
based on https://github.com/kubevirt/monitoring/pull/188, please rename the alert name to `KubeVirtHcoSingleStackIPv6`

Reply:
Done.